types/hls.js/index.d.ts

```diff
@@ -1714,10 +1730,20 @@ declare class Hls {
      * position of live sync point (ie edge of live position minus safety delay defined by hls.config.liveSyncDuration)
      */
     readonly liveSyncPosition: number;
+    /**
+     * get : array of audio tracks exposed in manifest
+     */
+    readonly audioTracks: Hls.AudioTrack[];
```

Comment:
Could you please add `HlsAudioTrack` and use it here?

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43335#issuecomment-603298347